polybar-scripts/network-openwrt-status/luci.py

```diff
@@ -0,0 +1,3 @@
+host = '192.168.8.1'
```

Comment:
Same as above: Would it make sense to add the configuration to the same file? So that you only have to manage a single script?